projects/openapi-cli/src/captures/streams/sources/har.ts

```diff
@@ -84,8 +85,15 @@ export class HarEntries {
 
       let requestBodyBuffer = interaction.request.body.buffer;
       let requestContentType = interaction.request.headers['content-type'];
-      let requestBodyEncoded =
-        requestContentType && requestBodyBuffer.toString('base64');
+
+      const requestBodyEncoded =
+        requestContentType &&
+        (await toBase64(
```

Comment:
`content-encoding` could be an [array of strings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding).

Asserting the type here lets `toBase64` crash implicitly, which doesn't feel right. It would be cleaner to let the `toBase64` function accept a list of strings as input and explicitly throw in the event of a list being received, with an appropriate error message.

Then later we can update `toBase64` to support a list when we need it.

Reply:
Oh interesting. I thought that was just the header type -- didn't realize it was part of HTTP to apply the encoding in the order of the array. 

I'll look into this more tonight. 

Separately there's the discussion of whether we want encoding in our HARs. 
Right now the capture flow decodes the message before writing it to disk. There are tradeoffs (some information is lost) and it makes marginally slower?? but verify faster. 

We could also just save the bodies as-is and do the decoding on the read side (possibly being unable to decode some messages)